internal/services/access_flow.go

```diff
@@ -543,13 +582,8 @@ func convertAttributeFiltersApiToTerraformModel(ctx context.Context, filter mode
 	}, nil
 }
 
-func convertAttributeFiltersTerraformModelToApi(ctx context.Context, filter aponoapi.AttributeFilterTerraformV1) (*models.AttributeFilter, diag.Diagnostics) {
-	apiFilterNames, err := utils.ConvertInterfaceToListOfString(filter.GetAttributeValue())
-	if err != nil {
-		diagnostics := diag.Diagnostics{}
-		diagnostics.AddError("Client Error", fmt.Sprintf("Failed to convert attribute names for attribute filter: %s", err.Error()))
-		return nil, diagnostics
-	}
+func convertAttributeFiltersApiToTerraformModel(ctx context.Context, filter aponoapi.AttributeFilterTerraformV1) (*models.AttributeFilter, diag.Diagnostics) {
```

Comment:
does this work well with the grantess as well?

Reply:
yes